### DISCUSSION TITLE
How to cancel a selection in a dropdown selection box

### DISCUSSION THREAD

**Discussion by xiaozhiagi** [zauberzeug/nicegui#1365] [Q&A]
### Question

During the process of using ui.select, I wanted to cancel the selected content, but I didn't see the closing x operation at the far right. The current cancellation operation is cancelled through the backspace, so I hope to implement a close button with an x on the far right side

**Comment by falkoschindler** [comment]
Sorry, I don't understand what you mean. What should happen when you press such a "cancel" (or "close"?) button? What "selected content" do you mean? Can you give a code example and/or screenshot?
